source/extensions/filters/http/ext_proc/ext_proc.cc

```diff
@@ -1047,6 +1047,11 @@ void Filter::onFinishProcessorCalls(Grpc::Status::GrpcStatus call_status) {
 }
 
 void Filter::sendImmediateResponse(const ImmediateResponse& response) {
+  if (config_->isUpstream()) {
+    stats_.send_immediate_resp_upstream_ignored_.inc();
+    ENVOY_LOG(debug, "Ignoring send immediate response when ext_proc filter is in upstream");
```

Comment:
Hi, sorry for commenting on the old issue but what was the rationale for this? I actually hit this case, and would like to understand if this is the fundamental limitation of the envoy core level or it's just an extension side implementation matter. @yanjunxiang-google